specification/eventgrid/data-plane/Microsoft.AgFoodPlatform/stable/2018-01-01/AzureDataManagerForAgriculture.json

```diff
@@ -0,0 +1,2245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Schema of Azure Data Manager for Agriculture events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure Data Manager for Agriculture Configuration events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent.",
+    "version": "2018-01-01"
+  },
+  "paths": {},
+  "definitions": {
+    "AttachmentResourceType": {
+      "description": "Attachment resource type.",
+      "enum": [
+        "Party",
+        "Farm",
+        "Field",
+        "SeasonalField",
+        "Boundary",
+        "ApplicationData",
+        "HarvestData",
+        "TillageData",
+        "PlantingData",
+        "PlantTissueAnalysis"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "AttachmentResourceType",
+        "modelAsString": true
+      }
+    },
+    "BoundaryParentType": {
+      "description": "Boundary parent type enum.",
+      "enum": [
+        "Field",
+        "SeasonalField",
+        "Zone",
+        "Prescription",
+        "PlantTissueAnalysis",
+        "ApplicationData",
+        "PlantingData",
+        "TillageData",
+        "HarvestData"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "BoundaryParentType",
+        "modelAsString": true
+      }
+    },
+    "DataManagerForAgriApplicationDataChangedV2EventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.AgFoodPlatform.ApplicationDataChanged.V2 event.",
```

Comment:
@lmazuel  do we need to add "Preview" word in description or class name as discussed over the call ?